btcjson/chainsvrresults.go

```diff
@@ -870,3 +870,14 @@ type ProveUtxoChainTipInclusionVerboseResult struct {
 	HashesProven []string `json:"hashesproven"`
 	Hex          string   `json:"hex"`
 }
+
+// GetUtreexoProofVerboseResult models the data from the
+// getutreexoproof when the verbose flag is set.  When the
+// verbose flag is not set, just the hex-encoded string of the entire proof
+// is returned.
```

Comment:
Note: If the flag is omitted, it defaults to one (verbosity = 1)

Reply:
addressed